pkg/webreg/webreg.go

```diff
@@ -611,37 +611,85 @@ Example:
+</ul>
+
+<code>ci-operator</code> config:
+{{ yamlSyntax (index . "imagePromotionConfig") }}
+
+<h4 id="image-consumption">Consuming an Image</h3>
+<p>
+Once a repository is publishing an image for reuse by others, downstream users can
+configure <code>ci-operator</code> to use that image in tests by including it as a
+<code>base_image</code> or as part of the <code>tag_specification</code>. In general,
+images will be available as part of the <code>tag_specification</code> and explicitly
+including them as a <code>base_image</code> will only be necessary if the promoting
+repository is exposing them to a non-standard <code>ImageStream</code>. Regardless of
+which workflow is used to consume the image, the resulting tag will be available under
+the <code>stable</code> <code>ImageStream</code>. The following <code>ci-operator</code>
+configuration imports a number of images:
+</p>
+<ul>
+  <li>the <code>stable:custom-scripts</code> tag, published as <code>myregistry.com/project/custom-scripts:latest</code></li>
+  <li>the <code>stable:component</code> and <code>:repo-{scripts|tests}</code> tags, by virtue of them being published under <code>ocp/4.4</code> and brought in with the <code>tag_specification</code></li>
+</ul>
+
+<code>ci-operator</code> config:
+{{ yamlSyntax (index . "imageConsumptionConfig") }}
+
+<p>
+Once the image has been configured to be an input for the repository's tests in the
+<code>ci-operator</code> configuration, either explicitly as a <code>base_image</code>
+or implicitly as part of the <code>tag_specification</code>, it can be used in tests
+in one of two ways. A registry reference step can be written to execute the shared tests
+in any <code>ci-operator</code> config, or a literal test step can be added just to one
+repository's configuration to run the shared tests. Two examples follow which add an
+execution of shared end-to-end tests using these two approaches. Both examples assume
+that we have the <code>ipi</code> workflow available to use.
+</p>
 
-<h4>Method 1: Adding a new reference:</h4>
+<h5>Adding a Reusable Test Step:</h4>
 
 <p>
-Make directory for repo: <code>ci-operator/step-registry/org/repo/setup/project</code>
+Full directions for adding a new resuable test step can be found in the overview for
+<a href="./adding-components#adding">new registry content</a>. An example of the process
+is provided here. First, make directory for the test step in the registry:
+<code>ci-operator/step-registry/org/repo/e2e</code>.
 </p>
 
-Make reference: <code>org-repo-tests-custom-ref.yaml</code>
+Then, declare a reusable step: <code>ci-operator/step-registry/org/repo/e2e/org-repo-e2e-ref.yaml</code>
 {{ yamlSyntax (index . "imageExampleRef") }}
 
-Make command: <code>org-repo-tests-integration-commands.sh</code>
+Finally, populate a command file for the step: <code>ci-operator/step-registry/org/repo/e2e/org-repo-e2e-commands.sh</code>
 {{ bashSyntax (index . "imageExampleCommands") }}
```

Comment:
not totally critical but how does this get into the container running the test image?  at what path? (the path might be of interest....)